Readme.md

```diff
@@ -47,45 +47,62 @@ simulations.  It is mainly developed at the Institute for
 
-You can find documentation on how to compile, use and develop ESPResSo
-on the homepage at [http://espressomd.org/html/doc/index.html](http://espressomd.org/html/doc/index.html).
+The [user guide](http://espressomd.org/html/doc/index.html) will
+walk you through the basic usage of ESPResSo. Advanced simulation
+methods are extensively documented, with examples and links to the
+relevant literature. Additional resources can be found on the
+homepage at http://espressomd.org/wordpress/documentation/, such
+as tutorials and doxygen documentation.
+
+## Installation
+
+Detailed installation instructions for Ubuntu and macOS can be found in the
+documentation, section [Installation](http://espressomd.org/html/doc/installation.html).
 
-ESPResSo is intended to be used by people that have proper knowledge
-of simulation techniques and know how to use them. We do not take
-responsibility if you use ESPResSo to create and publish bogus
-results. You have been warned!
+For most users, we recommend downloading the latest release version of ESPResSo. You
+can find it in the [release page](https://github.com/espressomd/espresso/releases),
+together with past releases until 4.0. When choosing a release, we recommend that
+you get the latest bugfix release in that line. For example, for 4.1 you would like
+to use 4.1.2.
 
-## PLEASE CITE US!
+### Please cite us!
 
-If you use ESPResSo and obtain scientific results that you publish, we
-would ask you to acknowledge the usage of ESPResSo by citing the relevant papers:
-http://espressomd.org/wordpress/about/please-cite-us/
+If you use ESPResSo to publish scientific results, we would ask you to
+acknowledge this usage by mentioning the software with its version number and
+[citing the relevant papers](http://espressomd.org/wordpress/about/please-cite-us/).
+A number of algorithms in ESPResSo are fairly advanced and unique to ESPResSo.
+The authors of these contributions kindly ask you to cite the relevant
+publications, as indicated in the documentation. For detailed instructions, see
+[How to cite ESPResSo](http://espressomd.org/html/doc/introduction.html#how-to-cite-espresso).
 
-A number of algorithms in ESPResSo are fairly advanced and unique to
-ESPResSo. The authors of these contributions kindly ask you to cite the
-relevant publications, as indicated in the ESPResSo User's Guide.
+### Join the community
+
+Please consider subscribing to our
+[mailing list](http://espressomd.org/wordpress/community-and-support/mailing-lists/)
+if you're actively using ESPResSo, as we occasionally need community
+feedback when making decisions on the future of specific features in
+upcoming releases. You'll also get notifications on bugfix releases.
 
 ## License
 
 Copyright (C) 2010-2019 The ESPResSo project
-Copyright (C) 2002,2003,2004,2005,2006,2007,2008,2009,2010 
-Max-Planck-Institute for Polymer Research, Theory Group
-  
+
+Copyright (C) 2002-2010 Max-Planck-Institute for Polymer Research, Theory Group
```

Comment:
I think we should just refer to the (non-existing) License file that could then also serve as the `header template`.

Reply:
I would be fine with shortening this section of the Readme to the following text, which links to the [license](https://github.com/espressomd/espresso/blob/python/COPYING) and acknowledges both the MPI and ESPResSo organizations:
```
Copyright (C) 2010-2019 The ESPResSo project
Copyright (C) 2002-2010 Max-Planck-Institute for Polymer Research, Theory Group
ESPResSo is free software: you can redistribute it and/or modify it under
the terms of the GNU General Public License as published by the Free
Software Foundation, either version 3 of the License, or (at your option)
any later version. A copy of the license is provided in [COPYING](COPYING).
```
The issue with providing instead a link to the [header template](https://github.com/espressomd/espresso/blob/python/maintainer/header_template.txt) is that the MPI line is missing.

Reply:
Why is the MPI part missing in the header template? I think we should move the header template to ```LICENSE``` in the root and add the MPI part.

Reply:
The license header can only contain years where the file received non-trivial contributions, therefore the MPI cannot appear for files created from scratch after 2010. Having this header template puts us in a grey area, since source files committed without a license header in the first commit where they appear are automatically licensed in a way that depends on the country(ies) of origin of the contributor, if I recall correctly. Adding the GPL after the fact during releases is not really permitted without the agreement of all people who committed non-trivial things in that file since its creation. That's why Google and Fedora have annoying bots that reject your contributions until you sign a legal document that allows them to relicense your contributions as GNU GPL if you forgot to put it in the first commit.

If I'm not mistaken, `COPYING` is the standard place to put the license in GNU GPL projects, while `LICENSE` is the standard place to put the license in Apache projects.